ipaserver/plugins/config.py

```diff
@@ -695,6 +701,23 @@ def pre_callback(self, ldap, dn, entry_attrs, attrs_list, *keys, **options):
                 raise errors.ValidationError(name=failedattr,
                     error=_('SELinux user map default user not in order list'))
 
+        if 'ipaconfigstring' in entry_attrs:
+            configstring = entry_attrs['ipaconfigstring']
+            if 'SubID:Disable'.lower() in map(str.lower, configstring):
```

Comment:
You can your new use is_config_option_present() function here.

Reply:
In pre-callback `entry_attrs` is coming from a synthetic entry built from args/options, not from LDAP, so I cannot use that function because the function actually looks up in a real LDAP entry.